theseus/geometry/so3.py

```diff
@@ -110,23 +115,26 @@ def _log_map_impl(self) -> torch.Tensor:
         sth = ret.norm(dim=1)
         theta = torch.atan2(sth, cth)
         # theta != pi
-        sel1 = 1 + cth > 1e-7
-        # Compute the approximation of theta / sin(theta) when theta ~ 0
-        scale1 = torch.where(
-            theta[sel1] >= 0.005, theta[sel1] / sth[sel1], 1 + sth[sel1] ** 2 / 6
+        not_near_pi = 1 + cth > 1e-7
+        # Compute the approximation of theta / sin(theta) when theta is near to 0
+        small_theta = theta[not_near_pi] < 5e-3
+        non_zero = torch.ones(1, dtype=self.dtype, device=self.device)
+        sth_nz = torch.where(small_theta, non_zero, sth[not_near_pi])
```

Comment:
Maybe `small_theta_nz`?

Reply:
`sth_nz` means `sine_non_zero`, not small `small_theta_non_zero`.

Reply:
Ah, my bad, I just quickly read from `torch.where` and mixed up the args (although tbh `small_theta_nz` doesn't make much sense, I guess). 

Can you rename to something more explicit? It'll make it easier if someone less familiar with Lie Groups needs to read this later (maybe `sine_theta`, `cosine_theta`)?.